test/spec/modules/gumgumBidAdapter_spec.js

```diff
@@ -46,6 +46,17 @@ describe('gumgumAdapter', function () {
       expect(spec.isBidRequestValid(bid)).to.equal(true);
     });
 
+    it('should return true when inslot sends sizes and trackingid', function () {
+      let bid = Object.assign({}, bid);
+      delete bid.params;
+      bid.params = {
+        'inSlot': '789',
+        'sizes': [[0, 1], [2, 3], [4, 5], [6, 7]]
+      };
+
+      expect(spec.isBidRequestValid(bid)).to.equal(true);
```

Comment:
I do not believe this tests the code change.

You'll need to probably build the requests `spec.buildRequests` and validate `data.bf` is present and looks correct.

Reply:
added a new unit test for this; thanks for calling that out :)